olive/systems/local.py

```diff
@@ -25,11 +28,17 @@ def run_pass(
         data_root: str,
         output_model_path: str,
         point: Optional[Dict[str, Any]] = None,
+        enable_fast_mode: bool = False,
```

Comment:
from the PR description, it seems the fast mode only enabled in local system. If that's the case, it would be better to move it to the `__init__`, In this way, all other system will not need be changed